src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/ConnectionButton.java

```diff
@@ -0,0 +1,128 @@
+package com.marginallyclever.makelangelo.plotter.plotterControls;
+
+import com.marginallyclever.communications.*;
+import com.marginallyclever.convenience.ButtonIcon;
+import com.marginallyclever.makelangelo.Translator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConnectionButton extends JPanel {
```

Comment:
ConnectionButton is not as descriptive as the old name.  This class is much more than a single button.

Reply:
It refers to the button which connect to the robot: 
![image](https://user-images.githubusercontent.com/23615562/147788477-a260e50f-d5ec-4393-b610-bc37f87eaa34.png)

What name do you prefer ?

Reply:
Ok, if it's just the button.  I see ChooseConnection.java was deleted so maybe I'm just confused.

Reply:
It's not just the button.  see comments elsewhere.

Reply:
I renamed it to ChooseConnection